core/blockchain.go

```diff
@@ -1276,9 +1277,10 @@ func (bc *BlockChain) writeKnownBlock(block *types.Block) error {
 
 	current := bc.CurrentBlock()
 	if block.ParentHash() != current.Hash() {
-		if err := bc.reorg(current, block); err != nil {
-			return err
-		}
+		// if err := bc.reorg(current, block); err != nil {
+		// 	return err
+		// }
+		return errors.New("Not saving block. Reorgs are disabled")
```

Comment:
externalize the error into a module variable, and reuse.

Also, i think linting expecte error messages to start with lowercase